### PR TITLE
Fix/Comment preprocess function is missing a space

### PR DESCRIPTION
**Pull request by xkopenreview**
image

this pr should add a space reader value and "must be readers..."

### CODE REVIEW

**Comment by melisabok** [comment]
some tests are failing here , can you check if it is related to these changes?

**Comment by xkopenreview** [comment]
it looks like in process function 
>${reader} 

can't be used because it will be recognized as json pointer
the tests are failing because i used ${reader} in string interpolation

need to take note of this if it's not fixable

**Comment by melisabok** [comment]
that's right, you need to use + to merge string.

File: openreview/venue/process/comment_pre_process.js [hunk 1]
```diff
@@ -2,23 +2,29 @@ async function process(client, edit, invitation) {
...
   const domain = res2.groups[0]
 
-  const commentMandatoryReaders = domain.content?.comment_mandatory_readers?.value || []
+  const commentMandatoryReaders =
+    domain.content?.comment_mandatory_readers?.value || []
   for (const m of commentMandatoryReaders) {
-    const reader = m.replace('{number}', forum.number)
-    if (!note.readers.includes(reader)) {
-      return Promise.reject(new OpenReviewError({ name: 'Error', message: reader + 'must be readers of the comment' }));
+    const mandatoryReader = m.replace("{number}", forum.number)
```
Comment (melisabok) [inline openreview/venue/process/comment_pre_process.js:20]: can you revert this change and just add a space to the line 21?
